SparkPost/2web2ui
# FE-1275 - Upgrade Cypress to version 6.1.0

[FE-1275](https://sparkpost.atlassian.net/browse/FE-1275)

### What Changed

- Upgrades Cypress to version 6.1.0
- Replaces uses of `should('not.be.visible')` with `should('not.exist')` in the vast majority of test cases -> [see the relevant portion of the migration guide](https://docs.cypress.io/guides/references/migration-guide.html#Non-existent-element-assertions)
- Swaps CI browser to Chrome instead of electron due to [a bug with newer versions of Cypress](https://github.com/cypress-io/cypress/issues/9208)
- As a result of ☝️ , uses `cypress/browsers` Docker image instead of generic Node image - this also allows us to remove some of the setup steps in our CI config that are handled by default with this image and helps us avoid manually installing Chrome as part of our CI pipeline
- Upgrades Review Badger to the latest release
- Changes path to Cypress artifacts when targeted for upload

### How To Test

- Verify Cypress tests continue to pass with new version of Cypress
- Verify the new version of Cypress runs locally after running `npm install`
- Verify in CircleCI that build artifacts match artifacts that are present on recent runs on the `master` branch. You can do so by comparing these two jobs:
  + https://app.circleci.com/pipelines/github/SparkPost/2web2ui/3496/workflows/338d9f2f-1f79-47b7-bb9f-a55634d021c1
  + https://app.circleci.com/pipelines/github/SparkPost/2web2ui/3443/workflows/2bfa4425-eaec-486e-8ac4-439e16deaaf2

### To Do
